hepcrawl/parsers/arxiv.py

```diff
@@ -36,6 +37,8 @@
 )
 RE_PAGES = re.compile(r'(?i)(\d+)\s*pages?\b')
 
+RE_DOIS = re.compile(r'[,;\s](?=\s*)')
```

Comment:
apologies for jumping in here, I have 2 comments: 
the name `RE_DOI` is misleading, since it doesn't match a DOI, instead it is supposed to match a `separator` between 2 DOIs
The regexp should be refined to specifically split between 2 DOIs and not at any `,` or `;`  I suggest `r'[,;\s]+(?=\s*10[.]\d{4,})'` 
```
In [1]: dois = ['10.7916/d8-bbr3-zv47,10.1016/j.nuclphysa.2021.122138;10.1016/j.nuclphysa.2020.122133 10.1103/PhysRevLett.119.231101, 10.1142/S0218271817430222']

In [2]: DOI_SEP = r'[,;\s]+(?=\s*10[.]\d{4,})'

In [3]: split_dois = ([re.split(DOI_SEP, doi) for doi in dois])

In [4]: split_dois
Out[4]: 
[['10.7916/d8-bbr3-zv47',
  '10.1016/j.nuclphysa.2021.122138',
  '10.1016/j.nuclphysa.2020.122133',
  '10.1103/PhysRevLett.119.231101',
  '10.1142/S0218271817430222']]
```

Reply:
Thank you, indeed this regex will be more precise.